README.md

```diff
@@ -24,9 +24,12 @@ dependencies: [
     .package(url: "https://github.com/OpenCombine/OpenCombine.git", from: "0.11.0")
 ],
 targets: [
-    .target(name: "MyAwesomePackage", dependencies: ["OpenCombine",
-                                                     "OpenCombineDispatch",
-                                                     "OpenCombineFoundation"])
+    .target(name: "MyAwesomePackage",
+    	    dependencies: ["OpenCombine",
+                           .product(name: "OpenCombineFoundation", 
+			   	    package: "OpenCombine"),
+                           .product(name: "OpenCombineDispatch",
+			   	    package: "OpenCombine")]),
```

Comment:
Please align the lines.

Reply:
Sorry, I'm not sure exactly you'd format the lines, is this better?
```
targets: [
    .target(name: "MyAwesomePackage",
    	    dependencies: ["OpenCombine",
                           .product(name: "OpenCombineFoundation", package: "OpenCombine"),
                           .product(name: "OpenCombineDispatch", package: "OpenCombine")]),
]
```